target/linux/ipq40xx/files/drivers/net/phy/ar40xx.h

```diff
@@ -182,8 +184,12 @@ struct ar40xx_mib_desc {
 #define   AR40XX_PORT_VLAN0_DEF_CVID_S		16
 
 #define AR40XX_REG_PORT_VLAN1(_i)		(0x424 + (_i) * 0x8)
+
+#ifndef CONFIG_IPQ40XX_PORT_ISOLATION
 #define   AR40XX_PORT_VLAN1_CORE_PORT		BIT(9)
 #define   AR40XX_PORT_VLAN1_PORT_TLS_MODE	BIT(7)
+#endif
```

Comment:
Don't wrap these macros. It won't end up in the compilation in case it is not used.